pkg/commands/entrypoints/entrypoints.go

```diff
@@ -306,6 +306,9 @@ func (c *EntrypointsCommand) writeOutput(cwd string, results []string) error {
 		results[k] = childPath
 	}
 
+	if results == nil {
```

Comment:
What do you think if we do this nil check at line 299, and `return nil` if `results == nil`?

Reply:
We still need to write it as json below, so we cant exit early.

What if we just initialize the initial slice to 0 length on line 192.

```go
	modifiedEntrypoints := make([]string, 0)
```

Reply:
I had considered that, but was conflicted because the Go style guide prefers initializing with a nil slice over empty slice. I suppose this could be an exception since we're encoding to JSON.